src/main/kotlin/moody/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,16 @@
+package moody.racingcar.domain
+
+class Cars(val cars: MutableList<Car>) : MutableList<Car> by cars {
```

Comment:
```suggestion
class Cars(private val cars: MutableList<Car>) : MutableList<Car> by cars {
```
외부에서 cars 변수에 접근하지 않으니, private으로 접근 변경자 수정해도 좋을 것 같아요 ㅎㅎ